samples/audio/sof/CMakeLists.txt

```diff
@@ -6,3 +6,9 @@ find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 project(sample_sof)
 
 target_sources(app PRIVATE src/main.c)
+
```

Comment:
remove this?

Reply:
Yes, should be removed, I added it because I wanted to include header from SOF but that includes too many conflicting headers, better keep those in modules/audio/sof in zephyr wrapper.